scss/_layout.scss

```diff
@@ -1,15 +1,21 @@
 header {
-    background-color: lightgray;
-    font-size: 2rem;
+    background-color: $color-header;
+    font-size: 2.5rem;
     display: flex;
     flex-direction: row;
     justify-content: center;
     align-items: center;
-    padding: 1rem;
+    padding: 1.5rem;
+    font-family: $font-header;
+    border-bottom: 2px solid $header-border;
+
+    @media screen and (max-width: $mobile-width) {
```

Comment:
I think it depends on personal preference, but I feel it's better to put all of media query for responsive design style in the bottom. If you put media query in each classname, it's hard to understand or maintain for future members. If you have any question, let me know 😄